src/ImGui.NET/Generated/ImGuiTextFilter.gen.cs

```diff
@@ -34,7 +34,11 @@ public void Destroy()
         {
             ImGuiNative.ImGuiTextFilter_destroy((ImGuiTextFilter*)(NativePtr));
         }
+#if NETSTANDARD2_1_OR_GREATER || NETCOREAPP2_1_OR_GREATER
         public bool Draw()
+#else
+        public bool Draw()
```

Comment:
can you check why is it doing it like this over here?

Reply:
Oh I didn't notice that. Will check when I'm on my PC again. Thanks!

Reply:
Fixed.